pkg/controllers/cassandra/service/testing/testing.go

```diff
@@ -0,0 +1,59 @@
+package testing
+
+import (
+	"testing"
+
+	casstesting "github.com/jetstack/navigator/pkg/controllers/cassandra/testing"
+	apiv1 "k8s.io/api/core/v1"
+)
+
+type FixtureFactory func(t *testing.T) *casstesting.Fixture
+type ServiceFactory func(f *casstesting.Fixture) *apiv1.Service
+
+func RunStandardServiceTests(t *testing.T, newFixture FixtureFactory, newService ServiceFactory) {
+	t.Run(
+		"service created",
+		func(t *testing.T) {
+			f := newFixture(t)
+			f.Run()
+			f.AssertServicesLength(1)
```

Comment:
Can we perform any checks to ensure the created service has the required ports/type etc. set? Or is this done elsewhere? 😄

Reply:
There are e2e checks for the service ports.